### ISSUE TITLE
Support iojs for $0 stripping

### ISSUE THREAD

**Issue by rstacruz** [bcoe/yargs#138]
[index.js:26](https://github.com/bcoe/yargs/blob/master/index.js#L26) has this code:

```js
if (~x.indexOf('node')) return;
```

this should be checking for `'iojs'` as well. Otherwise, you get this:

```
• ./bin/gh-annotate --help
Usage:
../../../.nvm/versions/io.js/v1.6.2/bin/iojs bin/gh-annotate <file>
... | ../../../.nvm/versions/io.js/v1.6.2/bin/iojs bin/gh-annotate
```

**Comment by bcoe** [comment]
@rstacruz https://github.com/bcoe/yargs/pull/139, does that branch do the trick for you?

**Comment by rstacruz** [comment]
yes!

**Comment by bcoe** [comment]
@rstacruz sorry for the slow turn around, this should now be fixed in `yargs@3.7.1`. please let me know if it works for you.
